src/libs/actions/Policy.js

```diff
@@ -112,6 +112,15 @@ function getPolicyList() {
         });
 }
 
+/**
+ * Does the user have any (free policy) workspaces?
+ *
+ * @returns {Number}
+ */
+function hasWorkspaces() {
+    return _.size(_.filter(allPolicies, policy => policy && policy.type === CONST.POLICY.TYPE.FREE && policy.role === CONST.POLICY.ROLE.ADMIN)) > 0;
```

Comment:
This feels much better, but one last minor correction 😬 

Since we only want to test if there is at least one policy where your predicate is `true`, [underscore's `some`](https://underscorejs.org/#some) function is better in this case. Something like this:

```
function getWorkspaceCount() {
    return _.some(allPolicies, policy => policy && policy.type === CONST.POLICY.TYPE.FREE && policy.role === CONST.POLICY.ROLE.ADMIN);
}
```

Reply:
makes sense! 👍